src/System.Net.Primitives/src/System/Net/DecompressionMethods.cs

```diff
@@ -9,6 +9,7 @@ public enum DecompressionMethods
     {
         None = 0,
         GZip = 1,
-        Deflate = 2
+        Deflate = 2,
```

Comment:
Nit: Why not specify using binary (or hex) constants instead of decimal?

Reply:
I was just following suit on what was already done.  But I can change it to hex.